src/main/java/appeng/client/gui/implementations/GuiInterface.java

```diff
@@ -59,13 +67,18 @@ protected void addButtons() {
         this.BlockMode = new GuiImgButton(this.guiLeft - 18, this.guiTop + 8, Settings.BLOCK, YesNo.NO);
         this.buttonList.add(this.BlockMode);
 
-        this.UnlockMode = new GuiImgButton(this.guiLeft - 18,this.guiTop + 26 ,Settings.UNLOCK, LockCraftingMode.NONE);
+        this.UnlockMode = new GuiImgButton(this.guiLeft - 18, this.guiTop + 26, Settings.UNLOCK, LockCraftingMode.NONE);
         this.buttonList.add(this.UnlockMode);
 
         this.interfaceMode = new GuiToggleButton(this.guiLeft - 18, this.guiTop + 44, 84, 85, GuiText.InterfaceTerminal.getLocal(), GuiText.InterfaceTerminalHint.getLocal());
         this.buttonList.add(this.interfaceMode);
     }
 
+    protected void addLabel() {
+        this.lockReason = new GuiImgLabel(this.fontRenderer, guiLeft + 40, guiTop + 12, Settings.UNLOCK, LockCraftingMode.NONE);
```

Comment:
`initGui` call originating from `setWorldAndResolution` causes the label to be re-added. You should probably clear labels manually in the `initGui` override.

![java_Bcm6m4eNYl](https://github.com/user-attachments/assets/27f92512-39d1-43a4-b667-b5fdca2d490c)

Reply:
I understand, but this PR actually has a potential error. When in Result Lock, it will show waiting for Pattern Output. But it will not show what this interface is waiting for. Because I can't get the locked stack. This is a compromise solution

Can you give a real solution?

Reply:
Fixed with latest commit